de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/wizards/drop_files/DropFilesDialog.java

```diff
@@ -1,39 +1,65 @@
-package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards;
+package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards.drop_files;
 
 import java.io.IOException;
 import java.nio.file.FileAlreadyExistsException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 
 import org.eclipse.e4.core.contexts.IEclipseContext;
-import org.eclipse.jface.wizard.Wizard;
 
+import de.tu_bs.cs.isf.e4cf.core.stringtable.E4CFileTable;
 import de.tu_bs.cs.isf.e4cf.core.util.RCPMessageProvider;
 import de.tu_bs.cs.isf.e4cf.core.util.services.RCPImageService;
 import de.tu_bs.cs.isf.e4cf.parts.project_explorer.DropElement;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Alert.AlertType;
+import javafx.stage.Stage;
+import javafx.scene.control.DialogPane;
 
 /**
  * A wizard that handles drops from the system explorer to the project explorer,
  * if the drop contains a nested element (e.g. directory)
  */
-public class DropWizard extends Wizard {
+public class DropFilesDialog {
 
-	private ImportDirectoryPage copyOptionsPage;
 	private DropElement dropElement;
-
+	private Alert alert;
+	private ImportDirectoryPage copyOptionsPage;

```

Comment:
I dont get this variable name

Reply:
As to why it's named that way, don't ask me.  I don't really have a prolem with it, but if i have to change it, i am open to suggestions.

Reply:
would definitely change copyOptionsPage to `importDirectoryPage`, that is the left over name of a previously renamed file :D
since the `alert` is not really an alert but rather the window to provide the view i would suggest calling it something like `dropFilesDialogWindow` ?

Reply:
page was renamed, alert is still alert: i think the name fits, because the user is alerted that some input is needed, but i can change that if thats  the consensus.